### PR TITLE
Subsystem skeletons

### PR DESCRIPTION
**Pull request by Cowpow0987**
Added Subsystems and Commands without any logic to be implemented at a later date on other branches.

### CODE REVIEW

File: src/main/java/frc/robot/commands/StartIntake.java
Comment (ItsMajestiX) [inline src/main/java/frc/robot/commands/StartIntake.java:1]: This is probably better off as a RunIntake where the intake stops when the command ends.

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java
Comment (ItsMajestiX) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java:1]: This subsystem needs either a method to limit its power or to set the output of the motor(s) depending on what the purpose of `setIntakePower()` is.

**Review by ItsMajestiX — CHANGES_REQUESTED** [review]
This is a good base. However, there are some changes that I would like to see before this is merged, along with the file specific comments I made.

1. Methods that should take values as input in subsystems (setters) should have parameters included.
2. Classes and methods should be documented using [Javadoc](https://www.tutorialspoint.com/java/java_documentation.htm) to describe what they do., what their parameters are, and what (if anything) they return. This is especially useful for making clear the units a function accepts or returns.

**Review by ItsMajestiX — CHANGES_REQUESTED** [review]
Very good. This is almost there, just two more things:
1. Make sure to clarify what units the set setpoint functions on the arm and climber expect. I would recommend they be in the same units as the get position/velocity functions.
2. Add a setter and a getter for speed to the intake subsystem.

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java
Comment (Cowpow0987) [reply src/main/java/frc/robot/subsystems/IntakeSubsystem.java:1]: SetIntakePower() is setting the max power, so I added one method to set the RPM setpoint and another to get the current RPM

**Review by ItsMajestiX — APPROVED** [review]
Looks good for now. We can go into more detail when we start implementing these.

**Pull request merged**